home-application/src/main/java/com/softserveinc/ita/homeproject/application/quartz/triggers/DataRetrievalTrigger.java

```diff
@@ -0,0 +1,16 @@
+package com.softserveinc.ita.homeproject.application.quartz.triggers;
+
+import com.softserveinc.ita.homeproject.application.quartz.config.QuartzConfig;
+import org.quartz.JobDetail;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.quartz.SimpleTriggerFactoryBean;
+
+@Configuration
+public class DataRetrievalTrigger {
+    @Bean(name = "dataRetrievalTriggers")
+    public SimpleTriggerFactoryBean triggerMemberStats(@Qualifier("dataRetrieval") JobDetail jobDetail) {
+        return QuartzConfig.createTrigger(jobDetail, 60000, "Data retrieval job trigger");
```

Comment:
Is there any example of a particular job? It isn't clear from naming. I would like to see how for instance notifications job will be set.

Reply:
I will change the class names. These are really incomprehensible names.

Reply:
Yes, I'll make a working example.